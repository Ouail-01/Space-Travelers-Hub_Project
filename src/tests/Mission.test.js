import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configureStore';
import Missions from '../components/Mission';

const mission = renderer.create(
  <Provider store={store}>
    <Missions />
  </Provider>,
);

describe('Mission component test', () => {
  it('Check if the missions section are available', () => {
    expect(mission).toMatchSnapshot();
  });
});
