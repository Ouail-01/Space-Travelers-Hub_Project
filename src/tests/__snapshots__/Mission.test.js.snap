// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mission component test Check if the missions section are available 1`] = `
<table
  className="missionTable"
>
  <thead>
    <tr>
      <th>
        Mission
      </th>
      <th>
        Description
      </th>
      <th>
        Status
      </th>
      <th>
         
      </th>
    </tr>
  </thead>
  <tbody />
</table>
`;
